quick_tour/the_big_picture.rst

```diff
@@ -1,267 +1,182 @@
-most important concepts behind Symfony and explain how you can get started
-quickly by showing you a simple project in action.
+Start using Symfony in 10 minutes! Really! That's all you need to understand the
+most important concepts and start building a real project!
 
 If you've used a web framework before, you should feel right at home with
-Symfony. If not, welcome to a whole new way of developing web applications.
+Symfony. If not, welcome to a whole new way of developing web applications. Symfony
+*embraces* best practices, keeps backwards compatbility (Yes! Upgrading is always
+safe & easy!) and offers long-term support.
 
 .. _installing-symfony2:
 
-Installing Symfony
-------------------
+Downloading Symfony
+-------------------
 
-Before continuing reading this chapter, make sure to have installed both PHP
-and Symfony as explained in the :doc:`/setup` article.
+First, make sure you've installed `Composer`_ and have PHP 7.1.3 or higher.
 
-Understanding the Fundamentals
-------------------------------
+Ready? In a terminal, run:
 
-One of the main goals of a framework is to keep your code organized and
-to allow your application to evolve easily over time by avoiding the mixing
-of database calls, HTML tags and other PHP code in the same script. To achieve
-this goal with Symfony, you'll first need to learn a few fundamental concepts.
+.. code-block:: terminal
 
-When developing a Symfony application, your responsibility as a developer
-is to write the code that maps the user's *request* (e.g. ``http://localhost:8000/``)
-to the *resource* associated with it (the ``Homepage`` HTML page).
+    $ composer create-project symfony/skeleton quick_tour
 
-The code to execute is defined as methods of PHP classes. The methods are
-called **actions** and the classes **controllers**, but in practice most
-developers use **controllers** to refer to both of them. The mapping between
-user's requests and that code is defined via the **routing** configuration.
-And the contents displayed in the browser are usually rendered using
-**templates**.
+This creates a new ``quick_tour/`` directory with, at first, just *one* file:
+``composer.json``. But when the command finishes, you'll find a small, but
+powerful Symfony application:
 
-When you go to ``http://localhost:8000/app/example``, Symfony will execute
-the controller in ``src/Controller/DefaultController.php`` and
-render the ``templates/default/index.html.twig`` template.
+.. code-block:: text
 
-In the following sections you'll learn in detail the inner workings of Symfony
-controllers, routes and templates.
+    symfony-project/
+    ├─ .env
+    ├─ .env.dist
+    ├─ bin/console
+    ├─ composer.json
+    ├─ composer.lock
+    ├─ config/
+    ├─ public/index.php
+    ├─ src/
+    ├─ symfony.lock
+    ├─ var/
+    └─ vendor/
 
-Actions and Controllers
-~~~~~~~~~~~~~~~~~~~~~~~
+Where did these files come from? They came from recipes installed by Symfony Flex.
```

Comment:
I thin kit's too early to mention concepts like Flex and recipes. They run a "create-project" command and they got a project created. That's intuitive. If we add this mention to Flex/recipes, things complicate without a clear benefit.